src/main/java/calculator/SimpleCalculator.java

```diff
@@ -0,0 +1,47 @@
+package calculator;
+
+import java.io.IOException;
+import java.security.InvalidParameterException;
+
+public class SimpleCalculator {
+
+    public static int sum(int a, int b) throws Exception {
```

Comment:
why the method throws particularly `Exception.class` if we know that the possible throwable exception is `InvalidParameterException`?